package.json

```diff
@@ -130,6 +131,7 @@
     "google-translate-api": "2.3.0",
     "gulp": "^4.0.0",
     "gulp-tape": "0.0.9",
+    "htmlhint": "^0.11.0",
```

Comment:
I've been asked to include the exact version number a couple of times when adding a module - I'd check with the maintainers once. 
LGTM otherwise.

Reply:
Used same for stylelint previously. Will have to change that too if this is the case.

Reply:
Yeah, we prefer to use the exact working version at least for app's dependencies. @vsvipul can you change this and also change other dependencies version (gulp etc).

Reply:
Sure, will do..

Reply:
@akashnimare changed to follow exact versioning for all modules.